### PR TITLE
Add .travis.yml file and tests

### PR DESCRIPTION
**Pull request by david-martin**

### CODE REVIEW

**Comment by luigizuccarelli** [comment]
@david-martin reviewed

**Comment by jasonmadigan** [comment]
reviewed, similar comments to https://github.com/feedhenry-templates/helloworld-cloud/pull/25

**Comment by jasonmadigan** [comment]
re-reviewed

File: application.js [hunk 1]
```diff
@@ -41,7 +40,23 @@ function startWorker() {
     console.log("App started at: " + new Date() + " on port: " + port); 
   });
 
-};
+}
+
+// Utility function: handle workers exiting (which can happen cleanly or due to an error)
+function workerExitHandler(worker, code, signal) {
```
Comment (jasonmadigan) [inline application.js]: Assume this was moved because it was previously hoisted and JSHint was failing

Comment (david-martin) [inline application.js]: yes, jshint used before defined

**Pull request merged**